### PR TITLE
Improve silo example

### PR DESCRIPTION
**Pull request by blaisb**
# Description of the problem

- Silo example had very old results for the timing
- Performance of the code has severely improved since then

# Description of the solution

- Reran the example, fixed some parameters and updated the timing results
- Lowered the simulation time (40s was 10 too many)

# Comments

- The simulation time are very different now... It takes 13ish hours on 8 cores instead of the old 48 hours on 32 cores... So it seems the code is like... 4-8 times faster than before on these kind of cases...

### CODE REVIEW

File: doc/source/examples/dem/silo/silo.rst [hunk 1]
```diff
@@ -169,14 +169,14 @@ In this simulation, we need a stopper (floating wall) in the filling stage (0-4
 
 Running the simulation
 ----------------------
-This simulation can be launched by (in parallel mode on 32 processes):
+This simulation can be launched by (in parallel mode on 8 processes):
```
Comment (voferreira) [inline doc/source/examples/dem/silo/silo.rst:172]:
```suggestion
This simulation can be launched in parallel (8 processes) running:
```

File: doc/source/examples/dem/silo/silo.rst [hunk 2]
```diff
@@ -169,14 +169,14 @@ In this simulation, we need a stopper (floating wall) in the filling stage (0-4
...
 
 .. code-block:: text
 
-  mpirun -np 32 dem_3d silo-Golshan.prm
+  mpirun -np 8 dem_3d silo-Golshan.prm
 
 .. warning::
-	This example takes approximately 48 hours on 32 cores. This high computational cost is due to the long simulation time (40 s of real-time) and the high number of particles. The number of particles can be reduced to have a shorter simulation time. For example, the entire simulation could be run in less than an hour on an 8 cores processor if only 13 230 particles are inserted.
+	This example takes approximately 14 hours on 8 cores. This high computational cost is due to the long simulation time (30 s of real-time).
```
Comment (voferreira) [inline doc/source/examples/dem/silo/silo.rst:179]: Maybe use "computational time" instead of "computational cost"?

**Review by voferreira — APPROVED** [review]
There are still two parameters to be fixed. Good to go after changing the following in the .rst file:
Line 103: set dynamic contact search size coefficient = 0.8 -> 0.9
Line 105: set load balance frequency                  = 100000 -> 10000
Additionally, the `Lagrangian post-processing` is not mentioned in the .rst file, but I don't know if you need to add it.

**Comment by blaisb** [comment]
> There are still two parameters to be fixed. Good to go after changing the following in the .rst file: Line 103: set dynamic contact search size coefficient = 0.8 -> 0.9 Line 105: set load balance frequency = 100000 -> 10000 Additionally, the `Lagrangian post-processing` is not mentioned in the .rst file, but I don't know if you need to add it.

I don't think it's necessary
Fixed the other parameters though. Thanks for the keen eye!

File: examples/dem/3d-silo/silo-Golshan.prm [hunk 3]
```diff
@@ -17,19 +18,26 @@ end
 
 subsection restart
   set checkpoint = true
-  set filename   = sliding_restart
```
Comment (acdaigneault) [inline examples/dem/3d-silo/silo-Golshan.prm:20]: Why did you remove it? Why not just change the name?

**Review by acdaigneault — APPROVED** [review]
Everything looks good!

File: examples/dem/3d-silo/silo-Golshan.prm (hunk 3 above)
Comment (blaisb) [reply examples/dem/3d-silo/silo-Golshan.prm:20]: I think it's better to leave the default restart file name...
